types/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "mc-bomb-types"
+version = "4.0.2"
+authors = ["MobileCoin"]
+edition = "2021"
+license = "GPL-3.0"
+rust-version = { workspace = true }
+
+[dependencies]
+displaydoc = { version = "0.2", default-features = false }
+mc-crypto-keys = { path = "../mobilecoin/crypto/keys" }
```

Comment:
Cargo sort forced me to merge these into one group, how do i prevent it from doing that?

Reply:
i guess i can look at the other ci's that run this